src/core/errors.ts

```diff
@@ -86,3 +88,6 @@ export enum ErrorCodes {
    */
   RoomLifecycleError = 102105,
 }
+
+// eslint-disable-next-line @typescript-eslint/no-unsafe-enum-comparison
+export const errorInfoIs = (errorInfo: ErrorInfo, error: ErrorCodes): boolean => errorInfo.code === error;
```

Comment:
Should we typedocs this as it's exported and used elsewhere?

Reply:
Ah my bad, missed this one - fixed!